frontend_app_workspace_advanced/src/action.async.js

```diff
@@ -7,8 +7,8 @@ export const getWorkspaceDetail = (apiUrl, workspaceId) =>
     method: 'GET'
   })
 
-export const getWorkspaceMember = (apiUrl, workspaceId) =>
-  fetch(`${apiUrl}/workspaces/${workspaceId}/members`, {
+export const getWorkspaceMember = (apiUrl, workspaceId, showDisabledUser = false) =>
+  fetch(`${apiUrl}/workspaces/${workspaceId}/members?show_disabled_user=${showDisabledUser ? 1 : 0}`, {
```

Comment:
since the endpoint has a default value for show_disabled_user to false, I would prefer not putting any query param instead of setting it to 0